.github/workflows/iroha2-dev-pr.yml

```diff
@@ -22,6 +22,7 @@ env:
   WASM_TARGET_DIR: wasm/target/prebuilt
   TEST_NETWORK_TMP_DIR: /tmp
   NEXTEST_PROFILE: ci
+  PROFILE: profiling
```

Comment:
why is this not `dev` or `test`?

Reply:
also, it seems unused. you should use it to install iroha

Reply:
IMO you should use it for `cargo install iroha`. And you should set it to `test`

Reply:
but others may disagree on what profile to use. @s8sato @0x009922 ?

Reply:
you will discover then that you need to increase limits because tests will get flaky

Reply:
I have no strong opinions on this

Reply:
I'm of the opinion that whenever we're running tests we should run the `test` build

Reply:
but on the other hand tests will run slower. We should take that into consideration

Reply:
I don't have a very strong opinion on this too.

Speaking of the profile of `irohad` used by `iroha_test_network`, since the goal was to have a black-boxed environment close to what final users will have, it could be a point towards using `release` profile.